crates/wasmcloud-host/src/messagebus/handlers.rs

```diff
@@ -139,9 +295,9 @@ impl Handler<Initialize> for MessageBus {
                     let _ = target
                         .send(super::rpc_client::Initialize {
                             host_id,
-                            nc: Arc::new(nc),
+                            nc: nc,
                             ns_prefix: ns,
-                            bus: bus,
+                            bus,
```

Comment:
Nitpick: you can also leave the right hand side of the `nc` off too since you're not wrapping it in an Arc

Reply:
(I see you doing that in other places too)

Reply:
yeah it's annoying. the IDE is supposed to mark when you can elide the field name but nooooo ... none of my IDEs do that.